src/Place/Place.php

```diff
@@ -271,12 +213,8 @@ public static function importFromUDB2Actor(
         return $place;
     }
 
-    /**
-     * @param PlaceImportedFromUDB2 $placeImported
-     */
-    public function applyPlaceImportedFromUDB2(
-        PlaceImportedFromUDB2 $placeImported
-    ) {
+    public function applyPlaceImportedFromUDB2(PlaceImportedFromUDB2 $placeImported): void
```

Comment:
Shouldn't these `apply` methods be `protected` now that I think about it? 🤔 
In theory they should not be accessible outside of the aggregate, since they are used to update the internal state after an event is recorded or the aggregate is being reconstructed from history (in both cases the event goes through `apply` method on the parent class which delegates it to these methods).

Reply:
Yes

Reply:
At least, that's what I did here already: https://github.com/cultuurnet/udb3-php/pull/414/files#diff-d6f2a68e86ca236e15d47231eeea2206R378